examples/gno.land/r/gnoland/coins/coins.gno

```diff
@@ -69,69 +68,92 @@ func Render(path string) string {
 func renderHomepage() string {
 	return strings.Replace(`# Gno.land Coins Explorer
 
-This is a simple, readonly realm that allows users to browse native coin balances.
-Here are a few examples on how to use it:
+This is a simple, readonly realm that allows users to browse native coin balances. Check your coin balance below!
 
-- ~/r/gnoland/coins:balances/<address>~ - show full list of coin balances of an address
-	- [Example](/r/gnoland/coins:balances/g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5)
-- ~/r/gnoland/coins:balances/<address>?coin=ugnot~ - shows the balance of an address for a specific coin
-	- [Example](/r/gnoland/coins:balances/g1jg8mtutu9khhfwc4nxmuhcpftf0pajdhfvsqf5?coin=ugnot)
+<gno-form path="balances">
+	<gno-input name="address" type="text" placeholder="Address (e.g. g1... or cosmos1...)" />
```

Comment:
Doesn't support non-cosmos prefix addresses, for example `neutron1sdq8pldqyjfqe2vv...`